Warehouse.Persistence.EF/Configuration/Entities/OrderLinesEntityConfiguration.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using Warehouse.Domain.Entities;
+
+namespace Warehouse.Persistence.EF.Configuration.Entities;
+
+public class OrderLinesEntityConfiguration : IEntityTypeConfiguration<OrderLine>
+{
+    public void Configure(EntityTypeBuilder<OrderLine> builder)
+    {
+        builder.ToTable("OrderLines");
```

Comment:
ToTable method is used to specify the table name for the entity. You can specify both singular and plural forms of the table name as needed.
By default, Entity Framework Core uses the pluralized form of the entity class name as the table name. If you want to override this behavior and specify a custom table name, you can use the ToTable method as shown in the example.